#goal definition
geometry_msgs/PoseStamped pose
---
#result definition
float32 error
---
#feedback
float32 error
# current end-effector pose
geometry_msgs/PoseStamped ee
